lib/ansible/module_utils/azure_rm_common_ext.py

```diff
@@ -0,0 +1,195 @@
+# Copyright (c) 2019 Zim Kalinowski, (@zikalino)
```

Comment:
this will likely be a merge conflict once #56359 is merged

Reply:
@nitzmahone I will remove common file once firewall is merged